controllers/admin/AdminPsThemeCustoConfiguration.php

```diff
@@ -139,6 +139,14 @@ public function initContent()
     {
         parent::initContent();
 
+        if (Module::isInstalled('ps_mbo')) {
+            $selectionModulePage = $this->context->link->getAdminLink('AdminPsMboModule');
+        } else {
+            $selectionModulePage = $this->context->link->getAdminLink('AdminModulesCatalog', true, array('route' => 'admin_module_catalog'));
```

Comment:
`, true, array('route' => 'admin_module_catalog')` won't change anything to the result and can be safely removed.